wasm.md

```diff
@@ -636,6 +650,104 @@ Unlike labels, only one frame can be "broken" through at a time.
     rule <k> (return => .) ~> FR:Frame ... </k>
 ```
 
+Memory
+------
+
+When memory is allocated, it is put into the store at the next available index.
+Memory can only grow in size, so the minimum size is the initial value.
```

Comment:
Let's add a note that currently only one memory is allowed per module, which is enforced by our check that the `<memAddrs>` cell is the empty `.Map` when creating a `memory`.

Reply:
Done